### ISSUE TITLE
XML pretty pretty printing causes extension to raise exceptions

### ISSUE THREAD

**Issue by NonLogicalDev** [AnWeber/httpyac#821]
I was running HttpYak agains a basic local server that was outputting sample HTML/JSON data and noticed that
  * mismatching ContentType like `text/html` for JSON would cause the extension to raise exception
  * DocTypes like `<!doctype html />` also cause exceptions as pretty printer does not like it

I think both issues are due to the XML parser trowing exceptions:

https://github.com/AnWeber/httpyac/compare/6.16.0...6.16.1#diff-7f25891c1de6aa3847cf8cb9c1bbf5b6f7e512662b18a71d5a236061a9ed140dR23-R26

**Comment by AnWeber** [comment]
That's right. It's bad when you can only nod when someone explains a bug. I will fix it. It came in due to the changed behavior of xmldom. I noticed it, but didn't think about it any further

**Comment by AnWeber** [comment]
if mimetype does not match content, the exception is caught and only logged. In this case thie body will not be changed to allow simpler debugging
